### PR TITLE
fixed confusing instructions

### PR DESCRIPTION
**Pull request by giladgressel**
changes made:

OLD: - `Each class should have an __init__ method that sets attributes
    to a default value if values are not passed.  `
This is confusing because you don't _need_ to have default values, you could have required values as well.

OLD: `- Each object should have at least three attributes.` -- this line removed
OLD: `- Each class should have at least two class attributes.`  -- this line removed. I don't believe that we are asking for class attributes? did we mean instance attributes? 

OLD: `Include a __str__ method in each class` combined with "create a print method" because that line of instruction seemed to be the same as the next line

I'm making this as a pull request because I'm not sure I understood the instructions myself entirely.
Did we want 3 instance attributes and 2 class attributes?
so 5 attributes total? 2 of which would be like

```
class car()
    thing = 0
    thing_ =1
    def __init__(self, thing1, thing2, thing3):
         self.thing1=thing1
         self.thing2=thing2
         self.thing3 = thing3
```
because that's technically what the instructions are currently asking for.
Let me know

### CODE REVIEW

File: 07_classes_objects_methods/07_03_freeform.py [hunk 1]
```diff
@@ -1,13 +1,9 @@
 '''
 - Write a script with three classes that model everyday objects.
-- Each class should have an __init__ method that sets attributes
-    to a default value if values are not passed.
+- Each class should have an __init__ method that sets at least three attributes
 - Create at least two objects of each class using the __init__ method.
-- Each object should have at least three attributes.
-- Each class should have at least two class attributes.
```
Comment (martin-martin) [inline 07_classes_objects_methods/07_03_freeform.py]: Yeah, I believe that Caden's idea was to also use class variables, not only instance variables, in this task.

File: 07_classes_objects_methods/07_03_freeform.py [hunk 2]
```diff
@@ -1,13 +1,9 @@
 '''
 - Write a script with three classes that model everyday objects.
-- Each class should have an __init__ method that sets attributes
-    to a default value if values are not passed.
+- Each class should have an __init__ method that sets at least three attributes
```
Comment (martin-martin) [inline 07_classes_objects_methods/07_03_freeform.py]: 👍

File: 07_classes_objects_methods/07_03_freeform.py [hunk 3]
```diff
@@ -1,13 +1,9 @@
...
 - Create at least two objects of each class using the __init__ method.
-- Each object should have at least three attributes.
-- Each class should have at least two class attributes.
-- Create a print method in each class that prints out the attributes
+- Include a __str__ method in each class that prints out the attributes
     in a nicely formatted string.
-- Include a __str__ method in each class.
```
Comment (martin-martin) [inline 07_classes_objects_methods/07_03_freeform.py]: 👍

**Comment by martin-martin** [comment]
Hei, Gilad, thanks for the PR and your thoughts! I agree right away with 2/3 of the changes you made. : )

I do think it could be good to get students to expose themselves to class variables. I also see that I haven't really written anything up for it in the course, so that will make it tough to hear about them in this lab like that.

However, I don't use class variables much myself, and I remember that students in earlier courses had some troubles in finding ideas for class variables.

Here's my suggestion:
* I'm going to add a course resource on class variables
* We ask them to add 1 class variable per class in this lab exercise

If you agree you can make the change and let me know. I'll post the resource when I have it up.

**Comment by giladgressel** [comment]
One of my students Simon stumbled upon a good reason to have a class variable -- if you want to count the number of instances that a class has created you can use a class variable and increment it.

He had a class of vehicles and he wanted to know how many vehicles were created.

In general the "make up your class" exercise is very contrived in my experience and hard for some students. I think it would also help if we gave them some ideas to work with in case they are having a hard time thinking up something to create.

But yea if you add resources on class variables, then this assignment would make a lot more sense.

**Pull request merged**